### PR TITLE
Update setup/README.md

### PR DESCRIPTION
**Pull request by JamesH65**
Some minor changes to a setup page that was rather out of date.

### CODE REVIEW

File: setup/README.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ A guide to setting up your Raspberry Pi
     - For keyboard layout configuration options see [raspi-config](../configuration/raspi-config.md).
 - [Power supply](../hardware/raspberrypi/power/README.md)
     - The Pi is powered by a USB Micro power supply (like most standard mobile phone chargers).
-    - You'll need a good-quality power supply that can supply at least 2A at 5V for the Model 3B, or 700mA at 5V for the earlier, lower powered models.
+    - You'll need a good-quality power supply that can supply at least 2A at 5V for the Model 3B and 3B+, or 700mA at 5V for the earlier, lower powered models. We recommend the official [Raspberry Pi power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/), which is designed specifically for all Raspberry Pi models.
     - Low current (~700mA) power supplies will work for basic usage, but are likely to cause the Pi to reboot if it draws too much power.
 
 ### Optional
 
 - Ethernet (network) cable [Model B/B+/2/3 only]
```
Comment (lurch) [inline setup/README.md]: Does this line also need updating, given that the Model 3A+ doesn't have Ethernet?

**Comment by lurch** [comment]
I edited the PR title to be more specific, because there are _lots_ of `README.md` files in this repo :wink:

File: setup/README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ A guide to setting up your Raspberry Pi
 ### Essential (for general use)
 
 - [SD Card](../installation/sd-cards.md)
-    - We recommend an 8GB class 4 SD card, ideally preinstalled with [NOOBS](../installation/noobs.md).
+    - We recommend a minimum of 8GB class 4, or class 10 SD card. This can be preinstalled with [NOOBS](../installation/noobs.md) or [Raspbian](../installation/installing-images/README.md) to save time, although setting up your own card is easy.
 - [Display and connectivity cable](monitor-connection.md)
     - Any HDMI/DVI monitor and any TV should work as a display for the Pi. For best results, use one with HDMI input, but other connections are available for older devices.
```
Comment (lurch) [inline setup/README.md]: Would "Any HDMI/DVI monitor and any TV" be better as "Any HDMI/DVI monitor or any TV" ?

File: setup/README.md (hunk 1 above)
Comment (JamesH65) [reply setup/README.md]: Yup

File: setup/README.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ A guide to setting up your Raspberry Pi
 
 ### Optional
 
-- Ethernet (network) cable [Model B/B+/2/3 only]
+- Ethernet (network) cable [Model B/B+/2/3B/3B+ only]
```
Comment (lurch) [inline setup/README.md]: For consistency, probably wouldn't hurt to change "2" to "2B"? :stuck_out_tongue_winking_eye:

Comment (JamesH65) [reply setup/README.md]: Wondered about that, but we didnt release a 2A, so not sure.

Comment (lurch) [reply setup/README.md]: I'll leave it up to the copy-editors.

**Pull request merged**